config.py

```diff
@@ -216,6 +216,7 @@ class Configuration(ConfigurationConstants):
     # Level 2 settings can be modified only by library managers and system admins (i.e. not by librarians).  Level 3 settings can be changed only by system admins.
     # If no level is specified, the setting will be treated as Level 1 by default.
     ALL_ACCESS = 1
+    global SYS_ADMIN_OR_MANAGER
```

Comment:
???

Reply:
This is used later on line 291. Without the `global` I get a `name 'SYS_ADMIN_OR_MANAGER' is not defined`. I wasn't sure about this because `SYS_ADMIN_ONLY` is used in a similar way but that doesn't throw an error.

Reply:
I removed this but moved the constant variables to `ConfigurationConstants` so I could call `ConfigurationConstants.SYS_ADMIN_OR_MANAGER`.

Reply:
Does it help to put these constants into `ConfigurationConstants`? They should probably be there anyway.